### ISSUE TITLE
Setting queueLimit causes System.FormatException

### ISSUE THREAD

**Issue by object** [serilog-contrib/serilog-sinks-grafana-loki#142]
### Which version of Serilog.Sinks.Grafana.Loki are you using?

v8.0.0

### Which version of .NET are you using?

net6.0

### Describe the bug

I tried to restrict a queue limit using the following code:

```
ILogger logger = new LoggerConfiguration()
    .WriteTo.GrafanaLoki(uri: "https://...", 
        credentials: new LokiCredentials
        {
            Login = "...",
            Password = "..."
        },
        labels: new List<LokiLabel>
        {
            new() {Key = "app", Value = "loki-memory-test"},
            new() {Key = "environment", Value = "test"}
        },
        queueLimit: 10)
    .MinimumLevel.Debug()
    .CreateLogger();
    SelfLog.Enable(Console.Error);

```
That causes this error to be logged to console:

> Caught exception while emitting to sink Serilog.Sinks.Grafana.Loki.LokiSink: System.FormatException: Input string was not in a correct format.
>    at System.Text.ValueStringBuilder.ThrowFormatError()
>    at System.Text.ValueStringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
>    at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
>    at Serilog.Debugging.SelfLog.WriteLine(String format, Object arg0, Object arg1, Object arg2)
>    at Serilog.Core.Sinks.SafeAggregateSink.Emit(LogEvent logEvent)
> 

If I remove the line "queueLimit: 10", then no such error occurs.

### To Reproduce

1. Create a console app
2. Add Serilog Grafana Loki Nuget package
3. Create a logger configuration using code specified in the error description
4. Run the app
5. Check console output

### Expected behavior

Specifying a valid queue limit should not cause System.FormatException

### Log/SelfLog output or exception with stacktrace

2022-10-13T14:43:06.3224669Z Caught exception while emitting to sink Serilog.Sinks.Grafana.Loki.LokiSink: System.FormatException: Input string was not in a correct format.
   at System.Text.ValueStringBuilder.ThrowFormatError()
   at System.Text.ValueStringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at Serilog.Debugging.SelfLog.WriteLine(String format, Object arg0, Object arg1, Object arg2)
   at Serilog.Core.Sinks.SafeAggregateSink.Emit(LogEvent logEvent)

### I have read the documentation

- [X] I have read [Common Problems](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems) section in project wiki

**Comment by mishamyte** [comment]
Hi @object,

Thank you for your report.
Will look into ASAP

**Comment by mishamyte** [comment]
Released in [v8.0.1](https://www.nuget.org/packages/Serilog.Sinks.Grafana.Loki/8.0.1)

@object, could you update and check, pls?

Also if I got you correctly, you are doing memory tests. I will be very grateful, if will have a possibility to see the results.
Because, have suggestions about the future of this project and want to keep this also in mind.

Thank you!

**Comment by object** [comment]
Thanks for the quick update, it works now!

Yes, we are doing some memory tests around your library because after we added logging to Grafana using your sink we experienced continuous growth of memory consumption in our services. We suspect it may have something to do with logging relatively large messages (more than 900 symbols). I need to gather more evidence, then I will submit another issue.
